DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -115,3 +115,19 @@
 
 
 #DQMOfflineHeavyIonsPhysics = cms.Sequence( dqmPhysics )
+
+##############################################################################
+# modifications of muon DQMOffline for pPb run
+def customiseRun2PPB_MuonDQMOffline(process):
+    # taget ptCut for Jpsi
+    process.globalAnalyzer.targetParams.ptCut_Jpsi = cms.untracked.double( 5.0)
+    process.trackerAnalyzer.targetParams.ptCut_Jpsi = cms.untracked.double(5.0)
+    process.tightAnalyzer.targetParams.ptCut_Jpsi = cms.untracked.double(  5.0)
+    process.looseAnalyzer.targetParams.ptCut_Jpsi = cms.untracked.double(  5.0)
+    # probe ptCoarse
+    process.globalAnalyzer.binParams.ptCoarse = cms.untracked.vdouble(  0.,1.,2.,3.,4.,5.,7.,9.,12.,15.,20.,30.,40.)
+    process.trackerAnalyzer.binParams.ptCoarse = cms.untracked.vdouble( 0.,1.,2.,3.,4.,5.,7.,9.,12.,15.,20.,30.,40.)
+    process.tightAnalyzer.binParams.ptCoarse = cms.untracked.vdouble(   0.,1.,2.,3.,4.,5.,7.,9.,12.,15.,20.,30.,40.)
+    process.looseAnalyzer.binParams.ptCoarse = cms.untracked.vdouble(   0.,1.,2.,3.,4.,5.,7.,9.,12.,15.,20.,30.,40.)
```

Comment:
@goni 
please add protecting ifs to apply modifications only if corresponding modules are known to the process
```
if hasattr(process, 'globalAnalyzer') and hasattr(process, 'trackerAnalyzer') and .. #or use individual if
   process.globalAnalyzer.targetParams.ptCut_Jpsi = cms.untracked.double( 5.0)
...
```